src/Services/StudyAreaService.ts

```diff
@@ -1009,6 +1010,9 @@ module StreamStats.Services {
 
         public selectGage(gage) {
             // selects gage and adds it to gage options for qppq
+            if () {

```

Comment:
If the applications includes Sediment, do the stuff for sediment. Else if the applications includes FDCTM, use the existing code in this function.